src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="big-search-form"
+      method="post"
+      name="big"
+      data-qa="big"
+    >
+      <label class="label-big-form">
+        <input
+          class="input-big-form"
+          type="text"
+          name="input-big-form"
```

Comment:
```suggestion
          name="inputBigForm"
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/217195011-06b49443-7d41-451c-a870-0019687666dc.png)